supplementary/devices/index.rst

```diff
@@ -633,8 +652,8 @@ Devices
      - IPS LCD (720x1280)
      - GPS, A-GPS, GLONASS, BDS
      - Ambient Light Sensor, Accelerometer, Proximity, Gyroscope
-     - IR Blaster
-   * - :doc: `Samsung Galaxy S7 <herolte>`
+     - IR Blaster 
```

Comment:
sorry for the nitpicking, but since you're touching it one more time anyway .... please remove the extra space at the end of the line